readme.md

```
@@ -103,6 +103,14 @@ You can fork or edit this source code !
   to [SwaggerEditor](https://editor.swagger.io).
 - Furthermore you can generate HTTP Client from this API using [openapi-generator](https://openapi-generator.tech/#try)
 
+## CURL for the api
+
+#### You need to encode to Base64 your user:pass [basic-auth] and pass in header
```

Markdownlint (0.37.0)

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)